client/events.go

```diff
@@ -24,6 +24,7 @@ func (cli *Client) Events(ctx context.Context, options types.EventsOptions) (<-c
 	started := make(chan struct{})
 	go func() {
 		defer close(errs)
+		defer close(messages)
```

Comment:
According to the documentation on this function, it's up to the caller to close the stream?

```
// Events returns a stream of events in the daemon. It's up to the caller to close the stream
```